.github/workflows/release-publish.yml

```diff
@@ -71,7 +96,7 @@ jobs:
 
   publish-to-docker-hub:
     name: Publish to DockerHub
-    needs: [publish-to-npm]
+    needs: [publish-to-npm, build-arm64]
```

Comment:
I'm probably missing something, but why do we need to depend on the arm64 step here? That step doesn't cache anything so isn't it a throwaway?